client/luigi-client.js

```diff
@@ -196,8 +196,8 @@
       return id;
     },
     /**
-     * Removes a context update listener
-     * @param {string} id the id that was returned by `addContextUpdateListener`
+     * Removes the context update listener.
```

Comment:
Same as above, it removes a specific listener, based on its id.

Reply:
Done.